tests/integration/test_counts.py

```diff
@@ -0,0 +1,115 @@
+import json
+from pathlib import Path
+
+import pytest
+
+from tests.integration import test_submit, utility_test_functions
+
+workflows_path = Path(__file__).parents[1].joinpath("workflows/")
+
+
+def check_status(test_workflow_id: str):
+    import time
+
+    count = 0
+    status = ""
+    print("Printing workflow status:")
+    while count < 10:
+        time.sleep(5)
+        count = 1 + count
+        # run status 1 time a minute unitl done, max 5x
+        status_result = utility_test_functions.run_cromshell_command(
+            command=["status", test_workflow_id],
+            exit_code=0,
+        )
+
+        status_result_formatted = json.loads(status_result.stdout)
+        status = status_result_formatted["status"]
+        print(status)
```

Comment:
should be taken out?

Reply:
I would prefer to keep it here for this test, It helps during debugging. It only prints if the test fails.

Reply:
then let's use `logging.DEBUG`.
Also, I'm not understanding why this only prints when a test fails.

Reply:
Tox/pytest will display any captured stdout when a test fails, so any print statements will be shown in this display. I just tried logging.DEBUG and INFO and neither of them are in the stdout capture. 

Example:
```
tests/integration/test_counts.py:113: AssertionError
------------------------------------------------------ Captured stdout call ------------------------------------------------------
Print workflow counts results:
4da4147a-b569-456b-b987-3f1e10e0b952    Failed
        HelloWorld.HelloWorldTask       0 Running, 0 Done, 0 Preempted, 1 Failed

```